tools/mzml_validator/mzml_validator.xml

```diff
@@ -0,0 +1,66 @@
+<tool id="mzml_validator" name="mzML Schema Validator" version="0.1.0+galaxy0" python_template_version="3.5" profile="21.05">
```

Comment:
remove python_template_version

Reply:
resolved in https://github.com/RECETOX/galaxytools/pull/319/commits/c5894768595b706f7de57feea61a4ee2e5585de7